workflow-manager/limiter/limiter.go

```diff
@@ -0,0 +1,69 @@
+package limiter
+
+import (
+	"fmt"
+	"sync/atomic"
+)
+
+// Limiter is similar to sync.Pool, however, it limtis how many shared
+// objects (in this case workers) there can be
+type Limiter struct {
+	workers    chan int32
+	inProgress int32
```

Comment:
We update `inProgress`as tickets come and go, but never use it, right? Is this here just so we could someday log or monitor how many tickets are in flight?

Reply:
Yeah, it's so we could potentially use them for debugging/monitoring.